util/archutil/Dockerfile

```diff
@@ -8,7 +8,9 @@ RUN apt-get update && apt-get --no-install-recommends install -y \
   binutils-s390x-linux-gnu \
   binutils-powerpc64le-linux-gnu \
   binutils-mips64el-linux-gnuabi64 \
-  binutils-mips64-linux-gnuabi64
+  binutils-mips64-linux-gnuabi64 \
+  binutils-loongarch64-linux-gnu \
```

Comment:
You need to change debian:bullseye-slim to debian:trixie-slim

Reply:
solved